.github/dependabot.yml

```diff
@@ -0,0 +1,21 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: '/'
+    schedule:
+      interval: daily
+    allow:
+      - dependency-type: 'production'
+    labels:
+      - 'prod-dependencies'
+
```

Comment:
What about adding docker actions pr?

Reply:
@chrisns Ping in case you missed this :)

Reply:
had planned to do that separately